tck/logs/src/main/java/org/eclipse/microprofile/telemetry/logs/tck/application/JulTest.java

```diff
@@ -49,13 +55,48 @@ public static WebArchive createDeployment() {
 
     private static final Logger julLogger = Logger.getLogger("jul-logger");
 
+    private static final String logFilePath = System.getProperty("log.file.path");
+
+    private static final String JUL_INFO_MESSAGE = "a very distinguishable info message";
+    private static final String JUL_WARN_MESSAGE = "a very distinguishable warning message";
+
+    private static final String SYS_OUT = "SystemOut";
+
     @Test
-    void julTest() throws InterruptedException {
+    void julInfoTest() throws IOException {
+        julLogger.log(Level.INFO, JUL_INFO_MESSAGE);
+        try {
+            Assert.assertTrue(checkMessage(SYS_OUT + ".*" + "INFO" + ".*" + JUL_INFO_MESSAGE));
```

Comment:
remove SYS_OUT, but you'll need to add something else in the line to avoid matching with the logs from the runtime (since you actually want to match against the logs from OTel bridge).